packages/elements/src/internals/machines/sign-in/router.machine.ts

```diff
@@ -363,7 +363,11 @@ export const SignInRouterMachine = setup({
         ChoosingStrategy: {
           tags: ['route:choose-strategy'],
           on: {
-            'NAVIGATE.PREVIOUS': 'Idle',
+            'NAVIGATE.PREVIOUS': {
+              description: 'Go to Idle, and also tell firstFactor to go to Pending',
+              target: 'Idle',
+              actions: sendTo(({ self }) => self.getSnapshot().children['firstFactor']!, { type: 'NAVIGATE.PREVIOUS' }),
```

Comment:
What would happen if `children['firstFactor']` is undefined?

Reply:
This isn't the best way to handle this in this scenario. I used it as I was working on something else where the `children` approach made sense. I've updated it.

That said, technically, it wouldn't ever be `undefined` in this scenario as `firstFactor` is invoked in this state's parent.